docs/QUICKSTART.md

```diff
@@ -2,45 +2,46 @@
+With browser integration configured, KeePassXC automatically enters 
 username/password credentials into web page fields.
 
 ## Installing and Starting KeePassXC
 
 * [Download the native installer](https://keepassxc.org/download) and install 
 KeePassXC for your  Windows, macOS, or Linux computer in the usual way for your platform.
 * Open the KeePassXC application.
-* Create a new database and give it a Master Key that's used to unlock the database file. 
+* Create a new database and give it a master key that's used to unlock the database file. 
 This database holds entries (usernames, passwords, account numbers, notes) 
 for all your websites, programs, etc.
 * Create a few entries - enter the username, password, URL, and optionally notes about the entry.
 * KeePassXC securely stores those entries in the database.
 
 
-## To set up Browser Integration with KeePassXC:
+## Setting up Browser Integration with KeePassXC
 
 * *Within KeePassXC*, go to **Tools->Settings** (on macOS, go to **KeePassXC->Preferences**.) 
-* In **Browser Integration**, check `Enable KeePassHTTP server`
+* In **Browser Integration**, check **Enable KeePassHTTP server**
 Leave the other options at their defaults.
-* *In your default web browser,* install the KeePassHTTP-Connector extension/add-on. Instructions for [[Firefox]](https://addons.mozilla.org/en-US/firefox/addon/keepasshttp-connector/?src=api) or [[Chrome]](https://chrome.google.com/webstore/detail/keepasshttp-connector/dafgdjggglmmknipkhngniifhplpcldb?utm_source=chrome-app-launcher-info-dialog) 
+* *In your default web browser,* install the KeePassHTTP-Connector extension/add-on. Instructions for [Firefox](https://addons.mozilla.org/en-US/firefox/addon/keepasshttp-connector/?src=api) or [Chrome](https://chrome.google.com/webstore/detail/keepasshttp-connector/dafgdjggglmmknipkhngniifhplpcldb?utm_source=chrome-app-launcher-info-dialog) 
 * Click the KeePassXC icon in the upper-right corner. You'll see the dialog below. 
 * Click the blue Connect button to make the browser extension connect to the KeePassXC application. 
 <img src="./KeePassXC-Connect.png" height="200" alt="KeePassXC Connect dialog">
+
 * *Switch back to KeePassXC.* You'll see a dialog (below) indicating that a request to connect has arrived. 
-* Give the connection a name (perhaps `Keepass-Browsername`, any unique name will suffice) and click OK to accept it.
+* Give the connection a name (perhaps *Keepass-Browsername*, any unique name will suffice) and click OK to accept it.
 * This one-time operation connects KeePassXC and your browser.
 <img src="./KeePassXC-Accept-Button.png" height="200" alt="KeePassXC accept connection dialog">
 
-## To use Browser Integration
+## Using Browser Integration
 
-* *Within the KeePassXC application,* double-click the URL of an entry,
-or select it and type Ctl-U (Cmd-U on macOS).
+* *Within KeePassXC,* double-click the URL of an entry,
+or select it and type Ctl+U (Cmd+U on macOS).
```

Comment:
You missed the missing r here. ;-)

Reply:
Err... I'm blanking on this - "missing r"?

Reply:
Ctrl, not Ctl.

Reply:
D'oh. That's a Mac-ism, I think. (Or I blame my Mac-centric view.) Good catch.